packages/sanity/src/_internal/cli/actions/deploy/helpers.ts

```diff
@@ -36,33 +40,42 @@ export interface ActiveDeployment {
 
 export interface UserApplication {
   id: string
-  projectId: string
+  projectId: string | null
```

Comment:
might be worth adding it here just so it's there

```suggestion
  projectId: string | null
  organizationId: string | null
```